7.8.вложенные_циклы.часть_1/7.8.5.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+for i in range(n):
+    for i in range(2):         
```

Comment:
1. Ни в коем случае нельзя называть переменные вложенных циклов как переменные родительских циклов.
2. Эти переменные у тебя не используются, следовательно нужно назвать их `_`.
```suggestion
for _ in range(n):
    for _ in range(2):         
```